jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/ExplorerToolWindow.kt

```diff
@@ -106,7 +106,8 @@ class ExplorerToolWindow(private val project: Project) : SimpleToolWindowPanel(t
         object : DoubleClickListener() {
             override fun onDoubleClick(event: MouseEvent): Boolean {
                 val path = awsTree.getClosestPathForLocation(event.x, event.y)
-                (path?.lastPathComponent as? AwsExplorerNode<*>)?.onDoubleClick()
+                val selected = path?.lastPathComponent as? DefaultMutableTreeNode
```

Comment:
Why is this needed?

Reply:
`(path?.lastPathComponent as? AwsExplorerNode<*>)?.onDoubleClick()`
This doesn't call the onDoubleClick() from AwsExplorerNode. It goes to the onClick method from DoubleClickListener.